Change InnerSource in Action Image

clcoffey opened the pull request:
The current image contains a logo we have been asked to remove.
I updated the image with the updated logo list.

lenucksi commented on events/isc-spring-2020-agenda.md:
```diff
@@ -16,7 +16,7 @@ Example of time differences in other timezones for the first day of conference:
 
 
-### Agenda
+### Agenda (Click title to watch on our <a href="https://www.youtube.com/playlist?list=PLCH-i0B0otNQeYBH5QvNRBDA3CMrS9lL9">YouTube Channel</a>)
```
Assuming that's tested and works. If it doesn't we can still revert it...

clcoffey committed:
Change InnerSource in Action Image

The current image contains a logo we have been asked to remove.
I updated the image with the updated logo list.

claredillon reviewed (approved):
Looks good - thanks @clcoffey
